getting-started-publishing/publishing-intro.md

```diff
@@ -0,0 +1,45 @@
+---
+info_no: Side om "Komme i gang - Publisere beskrivelser/kataloger"
+description: Skal gi overordnet informasjon om hvordan publisere beskrivelser/kataloger til data.norge.no
+url: https://data.norge.no/getting-started/publishing-data
+note: Vurder å slå sammen med "Creating descriptions"
+url: https://data.norge.no/getting-started/about-publishing
+---
+
+# Lage og publisere beskrivelser
+
+I denne guiden vil du lære
+
+- hvordan lage beskrivelser,
+- hvordan publisere beskrivelsene til data.norge.no, og
+- om hvilke tilganger du trenger.
+
+Det er to måter du kan lage og publisere databeskrivelser:
```

Comment:
Duplikat info fra creating descriptions. Vi bør holde oss til én.

Reply:
Infoen her er på en måte relevant begge steder. Kan ev. splitte opp registreringsløsning og å lage RDF-grafer selv i to forskjellige sider, så kontekstene er tydelig adskilt. Mulig det lager ny duplikat-informasjon.